examples/platform/alerts_common_scenarios.py

```diff
@@ -0,0 +1,155 @@
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""
+This example shows how to use the Alerts API and relate Alerts to other objects such as Processes and Observations.
+
+The SDK documentation is published on Read The Docs.  An Alerts Guide is available there.
+https://carbon-black-cloud-python-sdk.readthedocs.io
+
+If you are using SDK version 1.4.3 or earlier, see the related example script, alert_v6_v7_migration.py,
+also in examples/platform and the Alert Migration Guide on Read the Docs on the changes that are needed.
+SDK v1.5.0 does have breaking changes.
+"""
+
+import sys
+import time
+import json
+from cbc_sdk import CBCloudAPI
+from cbc_sdk.platform import Alert, WatchlistAlert
+from cbc_sdk.platform import Device
+
+# To see the http requests being made, and the structure of the search requests enable debug logging
+import logging
+logging.basicConfig(level=logging.DEBUG)
+
+
+def main():
+    """This script demonstrates how to use Alerts in the SDK and common operations to link to related objects.
+
+    This example does not use command line parsing in order to reduce complexity and focus on the SDK functions.
+    Review the Authentication section of the Read the Docs for information about Authentication in the SDK
+    https://carbon-black-cloud-python-sdk.readthedocs.io/en/latest/authentication/
+
+    This is written for clarity of explanation, not perfect coding practices.
+    """
+    # CBCloudAPI is the connection to the cloud.  It holds the credentials for connectivity.
+    # To execute this script, the profile must have an API key with the following permissions.
+    # If you are restricted in the actions you're allowed to perform, expect a 403 response for missing permissions
+    # Permissions are set on Settings -> API Access -> Access Level and then assigned to an API Key
+    # Alerts - org.alerts - READ: For Alert searching and facets
+    # Alerts - org.alerts.tags - CREATE, READ, DELETE
+    # Search - org.search.events - CREATE, READ: For Process and Observation searches
+    # Device - device - READ: For Device Searches
+    # Alerts - org.alerts.close - EXECUTE:
+    # Alerts - org.alerts.notes - CREATE, READ, UPDATE, DELETE
+
+    api = CBCloudAPI(profile="YOUR_PROFILE_HERE")
+
+    # To start, get some alerts that have a few interesting criteria set for selection.
+    # All the fields that can be used are on the Developer Network
+    # https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/alert-search-fields/
+
+    # start by specifying Alert as the type of object to search
+    alert_list = api.select(Alert)
```

Comment:
I wouldn't call this `alert_list` as it's actually `alert_query`

Reply:
changed everywhere